src/files/extra_headers.conf

```diff
@@ -1,6 +0,0 @@
-add_header Strict-Transport-Security "max-age=31536000; includeSubdomains" always;
```

Comment:
These have all been moved to `src/files/nginx_conf` to remove this extra file, is that correct?

Reply:
Correct - I'm not a fan of having loads of includes when these config lines can just sit in the main nginx.conf file.